portfolio/src/main/webapp/script.js

```diff
@@ -145,4 +145,93 @@ function dark() {
     document.getElementById('poem_1').style.backgroundColor = 'black';
     document.getElementById('poem_2').style.backgroundColor = 'black';
     document.getElementById('poem_3').style.backgroundColor = 'black';
-}
\ No newline at end of file
+}
+
+var amplitude = 70;
+var left_pos = 600;
+var top_pos = -600;
+var pi = Math.PI;
+var inc = pi / 30;
+var params = new Map();
```

Comment:
Can you find a better name for `params`?

What does this map store? Whether a bubble has popped or whether a bubble is still inflated?

Why do you not use booleans here? You seem to only set values to either 0 or 1.

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers